pkg/taxonomy/catalog.structs.schema.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
what are these taxonomy files doing here? Any taxonomy file that was under manager/config can be safely deleted completely. The rest (in top level config/taxonomy) shouldn't be touched in this PR. Let @ritwikc and @eletonia arrange them.